### DISCUSSION TITLE
Does this plugin support unattended installation via non-HTTP sources?

### DISCUSSION THREAD

**Discussion by synergiator** [hashicorp/packer-plugin-vsphere#643] [Community Help] [answered]
Imagine you want to provision configuration files for an unattended installation.

Typically you go for HTTP but if this not an option (or shouldn't be your first choice), what are possible alternatives?

Is USB support implemented in this plugin?

**Comment by tenthirtyam** [comment, marked as answer]
`cd_content` and `floppy_content` are options other than HTTP.

But do keep in mind the support of the guest operating system. For example, for Windows,`cd_content` is the path. For Linux, kickstart with `http_content` is common, but `cd_content` and `floppy_content` can be use IF the guest OS supports boot from floppy (dropped by many) or a second attached CD-ROM.

Hope that helps.
Ryan

**Comment by synergiator** [comment]
thx
